### ISSUE TITLE
[Feature request]: Provide user with better options when multiple ref branches are available

### ISSUE THREAD

**Issue by gentlecolts** [flatpak/flatpak#4905]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a feature request that matches the one I want to file, without success.

### Suggestion

When installing flatpaks, a user may be presented with the option to choose between multiple versions of dependencies, such as:

```
Similar refs found for 'org.freedesktop.Platform.openh264' in remote 'flathub' (system)

   1) runtime/org.freedesktop.Platform.openh264/x86_64/2.0
   2) runtime/org.freedesktop.Platform.openh264/x86_64/19.08

Which do you want to use (0 to abort)? [0-2]:
```

As a user who is not installing a specific version of openh264, or in fact not even explicitly installing h264 in the first place, this message provides me with no means of making an informed decision.  Ideally, I'd like to just automatically grab the newest valid version, and not see this prompt at all, or at the very least be presented with a "default" suggestion, but as is I'm just making a blind guess going through this prompt

**Comment by richard96292** [comment]
I agree with the initial suggestion and find it important. What is even worse when running a command like this `flatpak install -y --noninteractive flathub org.freedesktop.Platform.VulkanLayer.MangoHud` (notice how both -y and --noninteractive options are set) in a bash script the user is still getting prompted to choose the runtime version:
```
Similar refs found for ‘org.freedesktop.Platform.VulkanLayer.MangoHud’ in remote ‘flathub’ (system):

   1) runtime/org.freedesktop.Platform.VulkanLayer.MangoHud/x86_64/21.08
   2) runtime/org.freedesktop.Platform.VulkanLayer.MangoHud/x86_64/22.08

Which do you want to use (0 to abort)? [0-2]:
```
I suggest adding something similar to what docker is currently doing. Basically to have a `latest` branch that will install the latest runtime version. IT would look something like that:
```bash
# instead of this:
flatpak install -y --noninteractive flathub org.freedesktop.Platform.VulkanLayer.MangoHud/22.08
# have this:
flatpak install -y --noninteractive flathub org.freedesktop.Platform.VulkanLayer.MangoHud/latest
```

Moreover, the original proposal by gentlecolts can be followed at least when installing with `-y` and `--noninteractive` options.

**Comment by beedell-roke** [comment]
@richard96292, [`issues/4192`](https://github.com/flatpak/flatpak/issues/4192#issue-838892024) is relevant.
